Include/cpython/pystate.h

```diff
@@ -170,8 +170,18 @@ struct _ts {
     PyObject **datastack_limit;
     /* XXX signal handlers should also be here */
 
+    struct {
+        int initialized;
+    } _preallocated;
 };
 
+#define _PyThreadState_INIT \
+    { \
+        ._preallocated = { \
+            .initialized = 1, \
+        }, \
+    }
+
```

Comment:
Can it be turned into a variant below?

```c
#define _PyThreadState_INIT { ._preallocated.initialized = 1 }
```

Reply:
Thanks for the tip.  What's the advantage?

Reply:
Actually, this is a minor ajustment done while it does not require a separate pull request.

More compact declaration is easier to grasp in a single eye swipe. Like *_PyThreadState_INIT is the preallocated's initialized set to one* versus *_PyThreadState_INIT is... the preallocated's initialized set to one... ah, that's it*. In isolation of few lines it sounds funny but when a programmer reads the whole file to assemble a picture what  pystate is and what it is capable of, they read in zigzags only.